grafana_client/elements/alertingprovisioning.py

```diff
@@ -52,14 +52,18 @@ def update_alertrule(self, alertrule_uid, alertrule, disable_provenance=False):
         r = self.client.PUT(update_alertrule_path, json=alertrule, headers=headers)
         return r
 
-    def update_rule_group_interval(self, folder_uid, group_uid, alertrule_group):
+    def update_rule_group_interval(self, folder_uid, group_uid, alertrule_group, disable_provenance=True):
```

Comment:
Is it intended to have `disable_provenance=True` here, while it is `disable_provenance=False` on all other occasions?

Reply:
Great catch, not sure how I got that one wrong